test/meuse_comp.jl

```diff
@@ -0,0 +1,54 @@
+@testset "Meuse R gstat compatibility" begin
+    """meuse_grid.csv constains (x, y) plus estimation/variance of log(zinc)
+    for global and local (25 samples)
+    """
+    datadir = joinpath(@__DIR__,"data")
+    data_meuse = readgeotable(joinpath(datadir,"meuse.csv"), coordnames=(:x,:y))
+    grid_meuse = readgeotable(joinpath(datadir,"meuse_grid.csv"), coordnames=(:x,:y))
+
+    dim = 2
+    tol = 0.001
```

Comment:
could you please pass the `tol` directly as `0.001` in the `atol` option of isapprox. I think we don't need a variable here far from the line of usage.

Reply:
It is matter of taste I guess, but I can do it.

Reply:
Yes, it is also less error-prone I guess. If we define the tolerance up there, and later on decide to change the tolerance at the bottom of the test, we will have an outdated variable hanging around in the code. I try to be very picky with these coding styles and structure to facilitate maintenance and save us time in the long run.

Reply:
I disagree with tol variable, as it is defined to asses the allowed tolerance that the test will use. If we decide latter to use different tol values for different tests, well, in that case, the programmer can use a different one. Maybe the variable can be named default_tolerance.

Reply:
I like the idea. The point is to define some standard that allows us to iterate easily in the future and avoid bugs while updating the code.